modules/gdscript/gdscript.cpp

```diff
@@ -1613,33 +1619,36 @@ bool GDScriptInstance::set(const StringName &p_name, const Variant &p_value) {
 				Variant name = p_name;
 				const Variant *args[2] = { &name, &p_value };
 
-				Callable::CallError err;
-				Variant ret = E->value->call(this, (const Variant **)args, 2, err);
-				if (err.error == Callable::CallError::CALL_OK && ret.get_type() == Variant::BOOL && ret.operator bool()) {
-					return true;
+				Callable::CallError ce;
+				Variant ret = E->value->call(this, (const Variant **)args, 2, ce);
+				if (ce.error != Callable::CallError::CALL_OK) {
+					return false; // Let's allow normal handling, since the property is not found, in fact.
```

Comment:
```suggestion
					return false; // Let's allow normal handling, since the property is not found.
```

Could we remove the `, in fact.` from the comments? I don't think it adds much!